README.md

```diff
@@ -1,2 +1,114 @@
-# woltk
-Web of Life Toolkit
+# Woltka
+
+**Woltka** (Web of Life Toolkit App), is a bioinformatics package for shotgun metagenome data analysis. It takes full advantage of, and it not limited by, the [WoL](https://biocore.github.io/wol/) reference phylogeny. It bridges first-pass sequence aligners with advanced analytical platforms (such as QIIME 2). Highlights of this program include:
+
+- gOTU: fine-grain community ecology.
+- Tree-based, rank-free classification.
+- Combined taxonomic & functional analysis.
+
+Woltka ships with a **QIIME 2 plugin**. [See here for instructions](woltka/q2).
+
+## Contents
+
+- [Installation](#installation)
+- [Example usage](#example-usage)
+- Details
+  - [Input files](doc/input.md)
+  - [Output files](doc/output.md)
+  - [Classification system](doc/classify.md)
+  - [Ordinal matching](doc/ordinal.md)
+  - [Stratification](doc/stratify.md)
+- Tutorials
+  - [gOTU analysis](doc/gotu.md)
+  - [Tree-based classification](#tree-based-classification)
+  - [Combined taxonomic & functional analyses](#combined-taxonomic--functional-analyses)
+- For users of
+  - [QIIME 2](woltka/q2), [Qiita](doc/app.md#qiita), SHOGUN
+  - Bowtie2, BWA, Minimap2
+  - BLAST, DIAMOND, VSEARCH
+- References
+  - [Command-line interface](doc/cli.md)
+  - [Parameter auto-decision](doc/auto.md)
+  - [Developer's guidlines](doc/dev.md)
+- [Notes](#notes)
+
+## Installation
+
+Requirement: Python 3.6 or above.
+
+```bash
+pip install git+https://github.com/qiyunzhu/woltka.git
+```
+
+After installation, launch the program by executing:
+
+```bash
+woltka
+```
+
+## Example usage
+
+Woltka provides several small test datasets under [woltka/tests/data](woltka/tests/data). To access them, [download](https://github.com/qiyunzhu/woltka/archive/master.zip) this GitHub repo, unzip, and navigate to this directory.
+
+One can execute the following commands to make sure that Woltka functions correctly, and to get an impression of the basic usage of Woltka.
+
+1\. gOTU table generation ([details](doc/gotu)):
+
+```bash
+woltka gotu -i align/bowtie2 -o table.biom
+```
+
+The input path, [`align/bowtie2`](woltka/tests/data/align/bowtie2), is a directory containing five Bowtie2 alignment files (SAM format, xzipped), each representing the mapping of shotgun metagenomic sequences per sample against a reference genome database.
```

Comment:
Can you include something like the following snippet to be very explicit about what the directory looks like?
```
align/bowtie2
├── S01.sam.xz
├── S02.sam.xz
├── S03.sam.xz
├── S04.sam.xz
└── S05.sam.xz
```

Reply:
For more info, this is the output of
```
tree align/bowtie2
```
run from the `woltka/tests/data` directory

Reply:
I added an inline description: "(`S01.sam.xz`, `S02.sam.xz`,... `S05.sam.xz`)". I think it makes the page (which is the main readme page) slimmer, comparing to showing the entire directory tree. Commit [66cfa9c](https://github.com/qiyunzhu/woltka/pull/1/commits/66cfa9cf115dcdf548fc4ef2baab951442c8a4d9).